Source/WTF/wtf/SystemTracing.h

```diff
@@ -35,6 +35,14 @@
 #include <sys/types.h>
 #endif
 
+#if USE(SYSPROF_CAPTURE)
+#include <cstdio>
+#include <map>
+#include <sysprof-capture.h>
+#include <wtf/Lock.h>
+#include <wtf/text/CString.h>
+#endif
```

Comment:
This is a leftover from an earlier version of this commit, right? None of these are still needed here?